ui/js/graphs.js

```diff
@@ -150,6 +156,7 @@ perf.controller('GraphsCtrl', [
           tip.css({ opacity: 1, left: tipPosition.left, top: tipPosition.top });
         }
       });
+     },400);
```

Comment:
So this is actually a bit of a different implementation than I originally intended: I originally thought we would would pop up the tooltip immediately, and just defer getting the revision information. However, I think I actually like this approach a little better.  I'd change "400" to "250" though. Also, there needs to be a space between the "," and the number.

Totally not necessary, but if you could figure out how to not pop up the tooltip after the timeout if the cursor has moved away from the datapoint in the interim, that would be even better. :)